src/app/pages/account/users/user-list/user-list.component.html

```diff
@@ -14,101 +14,89 @@
   </ix-page-title-header>
 </ng-template>
 
-<ix-entity-empty
-  *ngIf="isLoading$ | async; else loaded"
-  [conf]="loadingConfig"
-></ix-entity-empty>
-
-<ng-template #loaded>
-  <mat-card>
-    <mat-card-content>
-      <table
-        ix-table
-        matSort
-        matSortDisableClear="true"
-        [matSortActive]="defaultSort.active"
-        [dataSource]="dataSource"
-        [matSortDirection]="defaultSort.direction"
-        [multiTemplateDataRows]="true"
-        [fixedLayout]="true"
-      >
-        <ng-container matColumnDef="username">
-          <th *matHeaderCellDef mat-header-cell mat-sort-header>
-            {{ 'Username' | translate }}
-          </th>
-          <td *matCellDef="let user; dataSource: dataSource" mat-cell>
-            {{ user.username }}
-          </td>
-        </ng-container>
+<mat-card>
+  <mat-card-content>
+    <table
+      ix-table
+      matSort
+      matSortDisableClear="true"
+      ix-table-empty
+      [emptyConfig]="(isLoading$ | async) ? loadingConfig : (emptyOrErrorConfig$ | async)"
```

Comment:
It would be good to find a way to reduce boilerplate for having those configs in component file as well.